app/src/main/res/values/strings.xml

```diff
@@ -270,7 +270,7 @@
     <string name="sort_by_times_opened">Times opened</string>
     <string name="sort_by_mobile_data">Mobile data</string>
     <string name="pref_import_export_blocking_rules">Import/export blocking rules</string>
-    <string name="pref_import_export_blocking_rules_msg">Import/export rules, import external rules from Watt or Blocker.</string>
+    <string name="pref_import_export_blocking_rules_msg">Import rules exported from Watt or Blocker.</string>
```

Comment:
This gives the wrong impression to the users about the setting item. It should describe what the users are expected to find when they click on the item.

Reply:
It still gives user the wrong impression.

**Title:** Import/export blocking rules
**Summary:** Import rules exported from Watt or Blocker.

The above is just plain wrong.